lib/tasks/deployment/20211006154552_remove_orphaned_avis.rake

```diff
@@ -0,0 +1,22 @@
+namespace :after_party do
+  desc 'Deployment task: remove_orphaned_avis'
+  task remove_orphaned_avis: :environment do
+    puts "Running deploy task 'remove_orphaned_avis'"
+
+    avis = Avis.unscope(:joins)
+      .joins('LEFT JOIN dossiers ON dossiers.id = avis.dossier_id')
+      .where(dossiers: { id: nil })
```

Comment:
```suggestion
    avis = Avis.unscope(:joins)
       .left_outer_joins(:dossier)
      .where(dossiers: { id: nil })
```

Ceci diff, j'ai regardé le sql rendu et y'a une ptite différence. Ma version inclut le default scope du dossier `"dossiers"."hidden_at" IS NULL` . Ce qui veut dire qu'elle va dégager aussi tous les avis des dossiers hidden.  Je ne sais pas si c'est ce qu'on veut.

Reply:
vu en direct, le code fait ce qu'on veut